shared/teams/team/rows/index.tsx

```diff
@@ -266,13 +266,13 @@ export const useEmojiSections = (teamID: Types.TeamID): Array<Section> => {
     ),
   })
 
-  sections.push({
-    data: ['emoji-header'],
-    key: 'emoji-header',
-    renderItem: () => <EmojiHeader />,
-  })
-
   if (!waiting && customEmoji) {
```

Comment:
I think this means to be `customEmoji.length`

Reply:
oh yep you're right, good catch!